tests/test_unit.py

```diff
@@ -7542,6 +7542,47 @@ def append_sum(tokens):
             result, expected, msg="issue with OnlyOnce after reset"
         )
 
+    def testGoToColumn(self):
+        """tests for GoToColumn class"""
+
+        dateExpr = pp.Word(pp.nums + ".")("date")
+        realNum = pp.Word(pp.nums + ".")("real")
```

Comment:
Please use `real` and `integer` from `pyparsing_common` (or just use `pyparsing_common.number`), and change `dateExpr` to `Regex(r"\d\d(\.\d\d){2}")`. Defining reals as `Word(nums + '.')` is an anti-pattern that is listed in https://github.com/pyparsing/pyparsing/wiki/The-Classic-Blunders. It will match ".", "....", and many other strings that are not valid reals. I don't want to put out examples that will mislead those who might look through any of the test or example code for samples.

Reply:
Ah thank you! Good call, I don't want to introduce misleading examples.